src/toolbox_monkey_patch.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from 'blockly/core';
+
+Blockly.Toolbox.prototype.onKeyDown_ = function () {
```

Comment:
In this PR or in a followup (file an issue for it), you should make this only happen on install/uninstall instead of happening on load.

Reply:
Done, though just to check installation is a bit awkward since it needs to happen even before NavigationController is created--does this seem reasonable?